glib-macros/src/enum_derive.rs

```diff
@@ -29,11 +27,11 @@ fn gen_genum_values(
         let mut value_name = name.to_string().to_camel_case();
         let mut value_nick = name.to_string().to_kebab_case();
 
-        let attrs = parse_item_attributes("genum", &v.attrs);
+        let attrs = parse_item_attributes("enum_value", &v.attrs);
         let attrs = match attrs {
             Ok(attrs) => attrs,
             Err(e) => abort_call_site!(
-                "{}: GEnum enum supports only the following optional attributes: #[genum(name = \"The Cat\", nick = \"chat\")]",
```

Comment:
that should probably be called member

Reply:
I think `enum_value` is more consistent since it corresponds to creating a `GEnumValue`